pkg/services/ngalert/api/tooling/definitions/alertmanager.go

```diff
@@ -48,6 +48,14 @@ import (
 //       200: GettableUserConfig
 //       400: ValidationError
 
+// swagger:route GET /api/alertmanager/grafana/config/api/v1/raw alertmanager RouteGetGrafanaRawAlertmanagerConfig
+//
+// gets the alertmanager config as plain text
+//
+//     Responses:
+//       200: GettableRawAlertmanagerConfig
```

Comment:
I think the "raw" implies that we aren't necessarily returning structured JSON here, right? I think the fact that we're wrapping the raw value in a struct adds a layer of indirection here.  If a config is correct, you should be able to GET it and POST it back with no changes - the struct wrapping breaks this.

It also gets incredibly hard for a client to consume JSON inside a string inside JSON. The inner json will get escaped, so `"` gets mapped to `\"`, and so on.

Instead, we can have this route as returning free-form text, with `Content-Type: text/plain`. This feels the most appropriate given that it's a "raw" config file - it can also return raw text.